packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -229,6 +228,98 @@ describe("IntervalCollection fuzz testing", () => {
 	});
 });
 
+// currently fails
+describe.skip("IntervalCollection offline fuzz testing", () => {
+	const model: DDSFuzzModel<SharedStringFactory, Operation, FuzzTestState> = {
+		workloadName: "offline interval collection",
+		generatorFactory: () => take(100, makeOperationGenerator(defaultOptions)),
+		reducer:
+			// makeReducer supports a param for logging output which tracks the provided intervalId over time:
+			// { intervalId: "00000000-0000-0000-0000-000000000000", clientIds: ["A", "B", "C"] }
+			makeReducer(),
+		validateConsistency: assertEquivalentSharedStrings,
+		factory: new SharedStringFactory(),
+	};
+
+	createDDSFuzzSuite(model, {
```

Comment:
most of the config (and the model) here is currently the same. i'd love to codeshare more of the commonalities so that the differences are more obvious.